operator/src/main/resources/scripts/livenessProbe.sh

```diff
@@ -35,9 +35,9 @@ if [ ! "${DYNAMIC_CONFIG_OVERRIDE:-notset}" = notset ]; then
       tgt_file=$tgt_dir/$tgt_file             # add back in tgt dir path
       [ -f "$tgt_file" ] && [ -z "$(diff $local_fname $tgt_file 2>&1)" ] && continue  # nothing changed
       trace "Copying file '$local_fname' to '$tgt_file'."
-      cp $local_fname $tgt_file # TBD ignore any error?
+      copyIfChanged $local_fname $tgt_file
```

Comment:
Minor comment:  "copyIfChanged" exits with SEVERE if the copy or chmod fails. My assumption is that's the original reason why the function wasn't originally used here. Is this something we want to happen in the livenessprobe? IIRC, its copy is supposed to be more of a 'best effort'.

Reply:
I think I'm okay with the liveness probe failing and the server pod restarting when this copy fails. The other choice is that the server continues running with the old configuration for an unknown amount of time.

Reply:
Fine by me.